lib/scrabble.rb

```diff
@@ -14,5 +39,23 @@ def point_values
       "U"=>1, "V"=>4, "W"=>4, "X"=>8,
       "Y"=>4, "Z"=>10
     }
+
+  def scrabble_points
+      letters = upcase.split('')
+
+      total = 0
   end
+
+
+
+#First goal is to figure out a way to assign points based on the word being given to the progam. We'll need to define a method to connect each letter of a string to its assigned value in the method hash def point_values
```

Comment:
Too big for a first goal. First goal should be to score a single letter. Worry about words later. Words seems like it would be simple, but it's not our easiest case. Do the small small small things first and then build on them and let them change as you add more limitations/functionality.

1) Can I score a letter?
2) Can I score an empty string?
3) Can I score `nil`?
4) Can I score a word?
5) Now start worrying about the bonuses, etc.

I don't want you even thinking about step 2 before you make step one pass. Keep your blinders on and make the small thing work.